projects/step-frontend/src/lib/modules/timeseries/performance-view/filter-bar/filter-bar.component.scss

```diff
@@ -5,6 +5,13 @@
   flex-direction: column;
   gap: 16px;
   background: #f5f5f5;
+
+  ::ng-deep {
```

Comment:
As discussed here: https://trello.com/c/58KNICEQ/160-use-viewencapsulationfalse-instead-of-ng-deep we don't want to use :ng-deep

Reply:
@neogucky I know at some point it was mandatory to use :host on top of this. What is the workaround? I want to edit mat components just in one place